README.md

```diff
@@ -301,3 +301,103 @@ Pretty stnadard, we simply collect all the hashes and the associated types for t
 ### Evaluation
 
 As discusssed in the pinning section, evaluation is done on a fully typechecked and pinned `Expr`ession.
+
+## Developing Inferno scripts with VScode
+
+In a shell go to /inferno and run `nix build .#vscode-inferno-lsp-server`. Then in VScode press `ctrl+shift+P` and run `Install from VSIX`. In the window, navigate to /inferno/result and select `inferno-lsp-server.vsix`.
+
+Do the same after for the VSIX created using `nix build .#vscode-inferno-syntax-highlighting`.
+
+### Regular Inferno
+
+In a shell go to /inferno and run `nix build .#inferno-lsp-server`
+Run `ls -al result`
+Copy the nix/store ... directory to your clipboard. Open VScode, press ctrl + shift + P and search for Open User Settings. Search for Inferno,
+find the inferno LSP extension tab and open it. Paste the directory you copied into the `Path to the inferno-lsp-server executable` field. 
+
+Be sure to append `/bin/inferno-lsp-server` to the end of the directory, then restart VScode.
+
+Create a new file with the .inferno extension. If you begin typing an inferno command such as Array.argmax, the autocomplete box should appear.
+
+Next, if you have the inferno repo cloned into ~/inferno
```

Comment:
It might be better to move this (or similar) to the beginning of this section and then refer to `~/inferno` throughout. Just mention that if users clone the repo to another location, they will need to change `~/inferno` to wherever it was cloned

Reply:
Okay, I'll switch that.